osbenchmark/utils/io.py

```diff
@@ -296,38 +303,66 @@ def decompress(zip_name, target_directory):
     elif extension == ".bz2":
         decompressor_args = ["pbzip2", "-d", "-k", "-m10000", "-c"]
         decompressor_lib = bz2.open
-        _do_decompress_manually(target_directory, zip_name, decompressor_args, decompressor_lib)
+        _do_decompress_manually(
+            target_directory, zip_name, decompressor_args, decompressor_lib
+        )
```

Comment:
This reformatting here and a number of other places doesn't seem necessary. Are you using a plugin?